pkg/virt-v2v/conversion/conversion.go

```diff
@@ -150,6 +150,9 @@ func (c *Conversion) addVirtV2vVsphereArgs(cmd utils.CommandBuilder) (err error)
 	cmd.AddArg("-i", "libvirt").
 		AddArg("-ic", c.LibvirtUrl).
 		AddArg("-ip", c.SecretKey)
+	if c.HostName != "" {
+		cmd.AddArg("--hostname", c.HostName)
+	}
```

Comment:
NP: The `AddArg` is already checking if the variable is empty, otherwise it would break the key arg format.